_test/export.response.test.php

```diff
@@ -163,4 +165,39 @@ public function test_element_name_is_empty_when_page_has_no_title()
         $name = (string)$names[0];
         $this->assertEmpty($name, 'Expected name in XML should be empty when page has no title.');
     }
+
+    public function test_element_description_is_entire_page_data()
+    {
+        // Generate much text for DokuWiki page
+        $muchContent = '';
+        for ($i=0; $i <= 500; $i++){
```

Comment:
Try `str_repeat()`.

Reply:
Cool that there is a function for that. Will use it :)